lms/envs/common.py

```diff
@@ -1795,7 +1795,7 @@
     'certificates',
     'dashboard',
     'instructor',
-    'instructor_task',
+    'lms.djangoapps.instructor_task',
```

Comment:
This is due to a bug in the tests migrations setup. Has been fixed in https://github.com/edx/edx-platform/pull/11603.